src/constEval.ts

```diff
@@ -645,25 +702,28 @@ function interpretEscapeSequences(stringLiteral: string) {
     );
 }
 
+function lookupID(ast: AstId, ctx: CompilerContext): Value {
```

Comment:
```suggestion
function lookupName(ast: AstId, ctx: CompilerContext): Value {
```

this is to make it in one style with `eqNames`

btw, thanks for factoring out this as a standalone function: this looks much better than being inlined